apiary.apib

```diff
@@ -93,31 +129,60 @@ current session status.
             },
             "data": {
                 "id": "sessionid123",
-                "type": "session"
+                "type": "session",
+                "relationships": {
+                    "account": {
+                        "links": {
+                            "related": "https://example.com/session/account"
+                        },
+                        "data": {
+                            "id": "abc1234",
+                            "type": "account"
+                        }
+                    }
+                }
             },
-            "related": {
-                "account": {
-                    "links": {
-                        "related": "https://example.com/session/account"
-                    },
-                    "data": {
-                        "id": "abc1234",
-                        "type": "account"
-                    },
-                    "related": {
+            "included": [{
```

Comment:
`"indluced"` would not be sent for `GET /session`, right? Only if I do `GET /session?include=account`?